tests/testapps/MinApp/MinApp.vsimporter/MinApp-WinStore10/project.json

```diff
@@ -0,0 +1,17 @@
+{
+  "dependencies": {
+    "WinObjC.Frameworks": "*",
```

Comment:
@jaredhms it did for a normal app developer. This is not a normal app. The two leaf nodes with a "*" version is exactly what we want most external developers to use as it is the simplest way to have them stay up to date on the latest pre-release version.

We don't currently have a switch in VSImporter to do version swapping but it is something we know about and are working on. See https://github.com/Microsoft/WinObjC/issues/2199